src/many-ledger/src/storage.rs

```diff
@@ -264,88 +268,47 @@ impl LedgerStorage {
         }
     }
 
-    pub fn mint(
-        &mut self,
-        to: &Identity,
-        symbol: &Symbol,
-        amount: TokenAmount,
-    ) -> Result<(), ManyError> {
-        if amount.is_zero() {
-            // NOOP.
-            return Ok(());
-        }
-        if to.is_anonymous() {
-            return Err(error::anonymous_cannot_hold_funds());
-        }
-
-        info!("mint({}, {} {})", to, &amount, symbol);
-
-        let mut balance = self.get_balance(to, symbol);
-        balance += amount.clone();
-
-        self.persistent_store
-            .apply(&[(
-                key_for_account(to, symbol),
-                fmerk::Op::Put(balance.to_vec()),
-            )])
-            .unwrap();
-
-        let id = self.new_transaction_id();
-        self.add_transaction(Transaction::mint(
-            id,
-            self.current_time.unwrap_or_else(SystemTime::now),
-            *to,
-            symbol.to_string(),
-            amount,
-        ));
-
-        if !self.blockchain {
-            self.persistent_store.commit(&[]).unwrap();
-        }
-        Ok(())
-    }
-
-    pub fn burn(
-        &mut self,
-        to: &Identity,
-        symbol: &Symbol,
-        amount: TokenAmount,
-    ) -> Result<(), ManyError> {
-        if amount.is_zero() {
-            // NOOP.
-            return Ok(());
-        }
-        if to.is_anonymous() {
-            return Err(error::anonymous_cannot_hold_funds());
-        }
-
-        info!("burn({}, {} {})", to, &amount, symbol);
-
-        let mut balance = self.get_balance(to, symbol);
-        balance -= amount.clone();
-
-        self.persistent_store
-            .apply(&[(
-                key_for_account(to, symbol),
-                fmerk::Op::Put(balance.to_vec()),
-            )])
-            .unwrap();
-
-        let id = self.new_transaction_id();
-        self.add_transaction(Transaction::burn(
-            id,
-            self.current_time.unwrap_or_else(SystemTime::now),
-            *to,
-            symbol.to_string(),
-            amount,
-        ));
-
-        if !self.blockchain {
-            self.persistent_store.commit(&[]).unwrap();
-        }
-
-        Ok(())
-    }
+    // TODO: Re-enable this when Minting is implemented
```

Comment:
Actually you should remove the code entirely. Minting might never work this way.